### PR TITLE
Update file 1

### PR DESCRIPTION
**Pull request by Govardhan458**
this is my first full request

### CODE REVIEW

File: file 1 [hunk 1]
```diff
@@ -1,2 +1,2 @@
-employee name
-employee salary
+employee name  grvdn
```
Comment (saiswaroop172) [inline file 1:1]: keep full name

Comment (Govardhan458) [reply file 1:1]: oky bro

Comment (saiswaroop172) [reply file 1:1]: have you done

Comment (Govardhan458) [reply file 1:1]: yes

**Review by Govardhan458 — COMMENTED** [review]
i'm done with all changes

**Comment by saiswaroop172** [comment]
i will merge

**Comment by saiswaroop172** [comment]
have you merged already

**Comment by Govardhan458** [comment]
yaa

**Pull request merged**